packages/server/src/fhir/bulkdata.ts

```diff
@@ -46,7 +53,16 @@ bulkDataRouter.get(
   })
 );
 
-bulkDataRouter.delete('/export/:id', (req: Request, res: Response) => {
+bulkDataRouter.delete('/export/:id', async (req: Request, res: Response) => {
+  const { id } = req.params;
+  const bulkDataExport = await getExportResource(id);
+
+  if (bulkDataExport.status !== 'cancelled') {
+    bulkDataExport.status = 'cancelled';
```

Comment:
Should we do this if the job is already completed?

Anything is better than the current non-implementation, so I'm pretty open minded here.

Reply:
In the `asyncJobCancelHandler` we do a test for `accepted` status in the patch operation before setting status to `cancelled` FWIW

Reply:
This was a requirement of the test case: Inferno expects the cancelled export to return a 404 status, even if it was completed before the cancellation occurred